analytics/folder_stats.sql

```diff
@@ -0,0 +1,141 @@
+/* 
+This script contains two parts: 1. lay out folder structure of the targeted FOLDER_IDs;
```

Comment:
This SQL script contains these 2 parts 1/2 - I would separate them into different .sql files since they serve different purposes.

@thomasyu888 [Also posed the question to me on a SQL script I wrote](https://github.com/Sage-Bionetworks/snowflake/pull/31/files#r1545533636):
> This is great! What are your thoughts on creating a stored procedure? Similar to #27

I am going to be looking at creating a stored procedure for my script - And convert my `.sql` file into an example of using it.